.github/workflows/lint.yml

```
@@ -2,19 +1,32 @@
 
 jobs:
-  black:
-    name: Black
-    runs-on: ubuntu-latest
+  ruff:
+    name: "Ruff"
+    runs-on: "ubuntu-latest"
     steps:
-      - name: 📥 Checkout the repository
-        uses: actions/checkout@v2
+        - name: "Checkout the repository"
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 8

(indentation)